pontoon/base/views.py

```diff
@@ -741,6 +741,21 @@ def update_translation(request):
         })
 
 
+@utils.require_AJAX
+def update_tour_status(request):
+    """Perform quality checks and return a list of any failed ones."""
+    try:
+        tour_status = request.POST['tour_status']
+    except MultiValueDictKeyError:
+        raise Http404
```

Comment:
This shouldn't be a 404 but a 400 (Bad Request Error), with a message explaining that the param is missing. You could get that for "free" by using a django Form.